platform/ontology/api_test.go

```diff
@@ -176,3 +177,51 @@ func TestNormalize(t *testing.T) {
 		}
 	}
 }
+
+var (
+	ontBlock = Block{
+		Hash:    "168d35ae9333f1d53ee0c124b44d268701df001df1313b388d001a5808f66d01",
+		Height:  7677564,
+		TxnList: []Tx{blockTx},
+	}
+
+	blockTx = Tx{
+		TxnTime:     1580115134,
+		ConfirmFlag: 1,
+		TxnHash:     "736fab4fa13435f201bc90a43ca5cd8c324ec88d6048fedb136f267371daee39",
+		Height:      7677564,
+	}
+	testBlock = blockatlas.Block{
+		Number: 7677564,
+		ID:     "168d35ae9333f1d53ee0c124b44d268701df001df1313b388d001a5808f66d01",
+		Txs: []blockatlas.Tx{
+			{
+				ID:     "736fab4fa13435f201bc90a43ca5cd8c324ec88d6048fedb136f267371daee39",
+				Block:  7677564,
+				Status: blockatlas.StatusCompleted,
+				Date:   1580115134,
+				Coin:   coin.Ontology().ID,
+				Meta:   blockatlas.Transfer{},
+			},
+		},
+	}
+)
+
+func TestPlatform_normalizeBlock(t *testing.T) {
+	res := normalizeBlock(ontBlock)
+	actual, err := json.Marshal(&res)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expected, err := json.Marshal(&testBlock)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !bytes.Equal(actual, expected) {
```

Comment:
try to use this package to do the assertions

"github.com/stretchr/testify/assert"


example:
https://github.com/trustwallet/blockatlas/tree/master/platform/ethereum/api_test.go#L510

Reply:
Do not think that it is necessary

Reply:
yes! it's easier, cleaner, anyone can understand, and the error log is more debuggable

Reply:
it's just a suggestion to have the code cleaner, and we already use this package

Reply:
@EnoRage we use `github.com/stretchr/testify/assert` consistency for all asserts in our codebase.